src/py42/exceptions.py

```diff
@@ -153,7 +164,13 @@ def raise_py42_error(raised_error):
     HTTPError's response status code.
     """
     if raised_error.response.status_code == 400:
-        raise Py42BadRequestError(raised_error)
+        if json.loads(raised_error.response.text)[0]["name"] in [
```

Comment:
Since the response text comes back as a string, and the code is simply `400`, I didn't see a better approach than this. We want to capture it before a `Py42BadRequestError` is raised.

Reply:
Could we not just do what we are doing in `py42.services.detectionlists.__init__.py` merthod: `handle_user_already_added_errpr` ?

I may not understand the problem, but see that method if you haven't already.  It checks the bad request error (a py42 error) for the error text.

Reply:
That's a much better way of doing it! I've updated based on this design.